<?xml version="1.0"?>
<launch>

	<param name="use_sim_time" value="true"/>
	<node name="VPG" pkg="grasp_learning" type="VPG.py" output="screen">
	 	<param name="input_output_data" value="$(find grasp_learning)/stored_data/training_data/task_dynamics/two_tasks_linear_dynamics_lambda_0.51.txt	" />
		<param name="load_model" value="0" />
		<param name="model_name" value="model.ckpt" />
 		<param name="num_inputs" value="2" />
		<param name="num_outputs" value="2" />
		<param name="num_rewards" value="1" />
		<rosparam param="hidden_units">[2]</rosparam>
		<param name="learning_rate" value="0.001"/>
		<param name="subtract_baseline" value="0"/>
		<param name="batch_size" value="3" />
		<param name="discount_factor" value="0.99" />
		<param name="use_normalized_advantages" value="1" />
		<param name="relative_path" value="$(find grasp_learning)/stored_data/tested_data/" />
		<param name="read_params_from_file" value="1" />
		<param name="max_num_trials" value="3" />
		<param name="min_dist_norm" value="0.04" />
 	</node>
</launch>
